tardis/montecarlo/montecarlo_numba/__init__.py

```diff
@@ -0,0 +1,7 @@
+from llvmlite import binding
+binding.set_option("tmp", "-non-global-value-max-name-size=2048")
```

Comment:
Wondering what this is for? IIRC this patch against LLVM "fixed", well, prevented, it from being an issue https://github.com/numba/llvmlite/blob/master/conda-recipes/0001-Revert-Limit-size-of-non-GlobalValue-name.patch .

Reply:
good to know. Thanks - haven't worked on it in a while.